content/docs/gto/get-started.md

```diff
@@ -0,0 +1,170 @@
+model releases) and assign them to different deployment environments (testing,
+shadow, production, etc.). Git-native mechanisms are used, so you can automate
+the delivery of your software project with CI/CD, and adopt a GitOps approach in
+general.
+
+This Get Started will walk you through basic GTO concepts and actions you would
+like to do in the Artifact Registry.
+
+## Showing the current state
+
+Assuming GTO is already [installed](/doc/gto/install) in your active Python
+environment, let's clone the example repo:
+
+```cli
+$ git clone https://github.com/iterative/example-gto
+$ cd example-gto
+```
+
+This repo represents a simple example of Machine Learning Model Registry (that
+is just a special case of an Artifact Registry for us). Let's review it:
+
+```cli
+$ gto show
+╒══════════╤══════════╤════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev   │ #prod   │ #staging   │
+╞══════════╪══════════╪════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1 │ v3.0.0  │ v3.1.0     │
+│ segment  │ v0.4.1   │ v0.4.1 │ -       │ -          │
+│ cv-class │ v0.1.13  │ -      │ -       │ -          │
+╘══════════╧══════════╧════════╧═════════╧════════════╛
+```
+
+Here we have 3 models: `churn`, `segment` and `cv-class`. The latest versions of
+them are shown in the column named `latest`. The latest is selected as the one
+having the greatest [SemVer](https://semver.org).
+
+Model versions could be promoted to different stages. Here we have 3 of them:
+`dev`, `prod` and `staging`. When a model was never promoted to a stage, we see
+`-` in the field.
+
+## Registering versions and assigning stages
+
+GTO can [register version](/doc/gto/command-reference/register) of artifacts and
+[assign stages to them](/doc/gto/command-reference/assign). Both functionalities
+work in a similar way, so let's walkthough only one of them here.
+
+Let's assume the version `v0.1.13` of `cv-class` looks very promising, and now
+we want to promote it to `dev` to test it:
+
+```cli
+$ gto assign cv-class --version v0.1.13 --stage dev
+Created git tag 'cv-class#dev#1' that assigns stage to version 'v0.1.13'
+To push the changes upstream, run:
+    git push origin cv-class#dev#1
+```
+
+GTO created a Git tag with a special format that contains instruction to assign
+a stage to a version. We can push to Git repository to start the CI, but let's
+ensure that changed our Registry first.
+
+```cli
+$ gto show
+╒══════════╤══════════╤═════════╤═════════╤════════════╕
+│ name     │ latest   │ #dev    │ #prod   │ #staging   │
+╞══════════╪══════════╪═════════╪═════════╪════════════╡
+│ churn    │ v3.1.1   │ v3.1.1  │ v3.0.0  │ v3.1.1     │
+│ segment  │ v0.4.1   │ v0.4.1  │ -       │ -          │
+│ cv-class │ v0.1.13  │ v0.1.13 │ -       │ -          │
+│ awesome  │ v0.0.1   │ -       │ -       │ -          │
+╘══════════╧══════════╧═════════╧═════════╧════════════╛
+```
+
+The `gto show` output confirms our expectation.
+
+## Acting downstream
```

Comment:
Not sure about the title (do people know what "downstream" means or refers to?). Can we brainstorm a little?

- React to stage assignments
- Integrate with GTO in CI/CD
- Artifact deployment